README.md

```diff
@@ -1,6 +1,10 @@
 Cyface Android SDK
 ========================
 
+[<img src="https://github.com/cyface-de/android-backend/workflows/Gradle%20Build/badge.svg">](https://github.com/cyface-de/android-backend/actions)
+[<img src="https://github.com/cyface-de/android-backend/workflows/Gradle%20Publish/badge.svg">](https://github.com/cyface-de/android-backend/actions)
+[![Build Status](https://app.bitrise.io/app/3f7da1f067e2d183/status.svg?token=Ggdr1abrQO7N_Pm1uI4kjg&branch=master)](https://app.bitrise.io/app/3f7da1f067e2d183)
```

Comment:
This token is only for this "batch icon" - it's generated automatically by Bitrise for that purpose only. No harm can be done with it.

Reply:
I think we should switch from Bitrise to Github Actions in the foreseeable future, like with all the other projects. As far as I noticed there are workflows for Android builds.

Reply:
agree